dmscripts/export_dos_opportunities.py

```diff
@@ -127,13 +127,13 @@ def upload_file_to_s3(
     logger,
 ):
     with open(file_path, 'br') as source_file:
-        acl = "public-read" if public else "private"
+        acl = "public-read" if public else "bucket-owner-full-control"
```

Comment:
Presumably there are no cases in which "private" might be appropriate here?

Reply:
I don't think so, at least not for the export-dos-opportunities script. This function `upload_file_to_s3` is used only in this module, so it's scope is pretty limited. It would be nice if there was a scripts repo wide `upload_file_to_s3` function, but I decided not to do that today...